db/mongodb_test.go

```diff
@@ -76,4 +78,31 @@ func TestMongoDB(t *testing.T) {
 	if metadata2 != "forty-two" {
 		t.Errorf("expected foruty-two as the answer, got %s", metadata2)
 	}
+	err = db.ExtraIndexes([]string{"teste.index1"})
+	if err != nil {
```

Comment:
Vamos usar a notação mais curta quando der ; )

```suggestion
	if err := db.ExtraIndexes([]string{"teste.index1"}); err != nil {
```